eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -25,12 +25,13 @@ public static function dumpAssets(CommandEvent $event)
         $options = self::getOptions($event);
         $appDir = $options['symfony-app-dir'];
         $webDir = $options['symfony-web-dir'];
-        $env = isset($options['ezpublish-asset-dump-env']) ? $options['ezpublish-asset-dump-env'] : '';
+        // @deprecated: Use of ezpublish-asset-dump-env is deprecated, use standard SYMFONY_ENV instead.
```

Comment:
But for DEV you don't dump assets anyway, do you?
As for us, we do use SYMFONY_ENV in dev as well - with containres and vms now we try to set up dev envs as close to prod as possible. We went as far as having a 'staging' env set up on our dev environment where eZ is accessed through nginx and varnish, so every single developer can troubleshoot caching problems :-)